testing/playwright/page-objects/NetworkMapsListPage.ts

```diff
@@ -1,19 +1,37 @@
 import { expect, type Page } from '@playwright/test';
 
+import { NavigationHelper } from '../utils/NavigationHelper';
 import { disableGuidedTour } from '../utils/utils';
 
 export class NetworkMapsListPage {
+  private readonly navigationHelper: NavigationHelper;
   private readonly page: Page;
 
   constructor(page: Page) {
     this.page = page;
+    this.navigationHelper = new NavigationHelper(page);
   }
 
-  async clickCreateNetworkMapButton() {
+  async clickCreateWithFormButton() {
     await this.page.getByRole('button', { name: 'Create network map' }).first().click();
     await this.page.getByTestId('create-network-map-form').click();
```

Comment:
nit: I think this test ID should read "dropdown-item" so its not mistaken for clicking on an actual form.